src/validations/user.validation.js

```diff
@@ -193,3 +193,48 @@ export const validateUserBody = (req, res, next) => {
 	}
 	next();
 };
+export const resetPassword = (req, res, next) => {
+	const schema = Joi.object({
+		password: Joi.string()
+			.regex(/^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#\$%\^&\*])(?=.{8,})/)
+			.required()
+			.messages({
+				'string.pattern.base': 'password should contain uppercase,lowercase,specialCharacter,and number',
+				'any.required': 'password is required',
+				'string.empty': 'password is not allowed to be empty'
+
+			}),
+		confirmPassword: Joi.string()
```

Comment:
can you check on the way of validating if it is equal to password without setting again this policies?

Reply:
done